libs/wingsdk/src/target-tf-aws/queue.ts

```diff
@@ -69,8 +67,9 @@ export class Queue extends cloud.Queue implements IAwsQueue {
     );
 
     const fn = new Function(
-      this.node.scope!, // ok since we're not a tree root
-      `${this.node.id}-SetConsumer-${hash}`,
+      // ok since we're not a tree root
+      this.node.scope!,
```

Comment:
i am curious why this is not just `this`...

can you raise an issue to follow up on this?

Reply:
to avoid cyclic dependencies I believe